docs/data/date-pickers/custom-layout/custom-layout.md

```diff
@@ -72,18 +72,20 @@ Use the `usePickerLayout` hook to get the subcomponents React nodes.
 Then you can fully customize the DOM structure.
 
 ```jsx
-const { usePickerLayout } from '@mui/x-date-pickers/PickersLayout';
+import { usePickerLayout } from '@mui/x-date-pickers/PickersLayout';
 
 const MyCustomLayout = (props) => {
-  const { toolbar, tabs, content, actionBar} = usePickerLayout(props);
+  const { toolbar, tabs, content, actionBar } = usePickerLayout(props);
 
   // Put the action bar before the content
-  return <div>
-    {toolbar}
-    {actionBar}
-    {content}
-  </div>
-}
+  return (
+    <PickersLayoutRoot>
+      {toolbar}
+      {actionBar}
+      {content}
```

Comment:
In a final example use would have to use `PickersLayoutContentWrapper` to wrap the content, but for the brevity of showing the idea it's probably better to use as minimal example as possible. 👌

Reply:
I don't know which one is better here
I fear that a misleading code snippet lead people to build broken apps :/

Reply:
Yes, that is my concern as well, especially when the longer example has the code collapsed.

Reply:
Do you prefer to keep as it is on the PR or to add `PickersLayoutContentWrapper` ?

Reply:
I'd vote for including it just for the sake of someone not expanding the complete demo and later complaining about having issues with using layout slot. 🙈

Reply:
Done